### ISSUE TITLE
Gulp watching `.temp` crash

### ISSUE THREAD

**Issue by IcanDivideBy0** [SamVerschueren/gulp-api-doc#2]
Hello,
I have the following tasks in my gulp file :

```
gulp.task('doc', function () {
  return gulp.src('lib/routes/**/*.js')
  .pipe(require('gulp-api-doc')())
  .pipe(gulp.dest('apidoc'));
});

gulp.task('doc:watch', function () {
  gulp.watch([
    'lib/routes/**/*.js',
    '!lib/routes/.temp'
  ], ['doc']);
});
```

Sometimes, gulp crashes with the following stack :

```
uncaughtException: ENOENT: no such file or directory, scandir '.../lib/routes/.temp/'
Error: ENOENT: no such file or directory, scandir '/home/samuel/Workspace/edito-api/lib/routes/.temp/'
    at Error (native)
    at Object.fs.readdirSync (fs.js:813:18)
    at Gaze.emit (/home/samuel/Workspace/edito-api/node_modules/gaze/lib/gaze.js:135:10)
    at Gaze.<anonymous> (/home/samuel/Workspace/edito-api/node_modules/gaze/lib/gaze.js:401:18)
    at null._onTimeout (/home/samuel/Workspace/edito-api/node_modules/gaze/lib/gaze.js:427:24)
    at Timer.listOnTimeout (timers.js:92:15)
```

It appears the folder is created from this module, then deleted, I can't find a way to make `gulp.watch` ignore it properly. Maybe it could be useful to overide the `tempPath` via an option ?

**Comment by SamVerschueren** [comment]
The `.temp` directory is not created by this plugin but probably by apiDoc itself.

Try `!lib/routes/{.temp, .temp/**}` as globbing pattern.

**Comment by SamVerschueren** [comment]
Oh you're right, it is created by this plugin. Sorry, haven't used it in a while. Will fix it.

**Comment by SamVerschueren** [comment]
Should be fixed in `1.1.0`.
